features/base/media/components/VideoTrack.js

```diff
@@ -29,12 +29,21 @@ class VideoTrack extends Component {
      */
     render() {
         const videoTrack = this.props.videoTrack;
+        let stream = null;
+
+        if (videoTrack) {
+            // XXX We may want not to start showing video until video stream
+            // has started to play anywhere else.
+            stream = this.props.waitForVideoStarted && !videoTrack.videoStarted
+                ? null
```

Comment:
If the VideoTrack is told to not waitForVideoStarted, it's not going to render the stream/videoTrack?